src/contracts/core/StakeRootCompendium.sol

```diff
@@ -485,9 +382,8 @@ contract StakeRootCompendium is StakeRootCompendiumStorage {
                 avsDirectory.getTotalAndAllocatedMagnitudes(operator, operatorSet, strategies);
 
             for (uint256 i = 0; i < strategies.length; i++) {
-                uint256 multipliedDelegatedShares = delegatedShares[i] * multipliers[i] / 1 ether;
-                delegatedStake += multipliedDelegatedShares * totalMagnitudes[i];
-                slashableStake += multipliedDelegatedShares * allocatedMagnitudes[i];
+                delegatedStake += delegatedShares[i] * totalMagnitudes[i] / 1 ether * multipliers[i];
```

Comment:
i *think* the previous saves more gas in the zkVM

Reply:
not sure though, we should benchmark

Reply:
ahh okay this was inadvertent